### PR TITLE
Fix development doc

### PR DESCRIPTION
**Pull request by chanseokoh**
@meltsufin

### CODE REVIEW

**Review by miraleung — APPROVED** [review]

File: DEVELOPMENT.md [hunk 1]
```diff
@@ -15,13 +15,13 @@
...
 
 2.  Copy the protos from Showcase into googleapis/google/showcase.
 
     ```sh
-    cp gapic-showcase/schema/google/showcase/v1beta1 googleapis/google/showcase/v1beta
+    mkdir googleapis/google/showcase
+    cp -r gapic-showcase/schema/google/showcase/v1beta1 googleapis/google/showcase/v1beta1
     ```
 
 3.  Add the new microgenerator rules to the protobuf directory's `BUILD.bazel`
```
Comment (meltsufin) [inline DEVELOPMENT.md:27]: I think it's better to clarify which file needs to be modified.
```suggestion
3.  Add the new microgenerator rules to `googleapis/google/showcase/v1beta/BUILD.bazel`
```

**Review by meltsufin — COMMENTED** [review]
For Step 4, I would also clarify that building the target should be from the `googleapis` project/directory.

File: DEVELOPMENT.md (hunk 1 above)
Comment (chanseokoh) [reply DEVELOPMENT.md:27]: Good idea. Note it's `v1beta1`, not `v1beta`, which is what I'm fixing in this PR too.

Comment (meltsufin) [reply DEVELOPMENT.md:27]: Right, sorry, fixed.

**Comment by chanseokoh** [comment]
@miraleung could you approve test workflows?

**Pull request merged**